pkg/utils/ocm.go

```diff
@@ -23,26 +23,33 @@ const (
 	stagingURL                 = "https://api.stage.openshift.com"
 	integrationURL             = "https://api.integration.openshift.com"
 	productionGovURL           = "https://api-admin.openshiftusgov.com"
+	integrationGovURL          = "https://api-admin.int.openshiftusgov.com"
+	stageGovURL                = "https://api-admin.stage.openshiftusgov.com"
```

Comment:
Be consistent with other variables naming conventions.
```suggestion
	stagingGovURL                = "https://api-admin.stage.openshiftusgov.com"
```